oci/layout/oci_dest.go

```diff
@@ -173,7 +173,7 @@ func (d *ociImageDestination) PutManifest(m []byte) error {
 	}
 
 	annotations := make(map[string]string)
-	annotations["org.opencontainers.ref.name"] = d.ref.tag
+	annotations["org.opencontainers.image.ref.name"] = d.ref.tag
```

Comment:
actually, I'd rather use what's defined in oci image-spec itself, I'll follow up on this.